provisioning/setup.yaml

```diff
@@ -1,22 +1,22 @@
 hare:
   post_install:
-    cmd: /opt/seagate/cortx/hare/bin/hare_setup
-    args: --report-unavailable-features --configure-logrotate --post_install
+    cmd: /opt/seagate/cortx/hare/bin/hare_setup post_install
+    args: --report-unavailable-features --configure-logrotate
   init:
-    cmd: /opt/seagate/cortx/hare/bin/hare_setup
-    args: --init
+    cmd: /opt/seagate/cortx/hare/bin/hare_setup init
+    args:
   config:
-    cmd: /opt/seagate/cortx/hare/bin/hare_setup
-    args: --config $URL --filename '/var/lib/hare/cluster.yaml'
+    cmd: /opt/seagate/cortx/hare/bin/hare_setup config
+    args: --config $URL --filepath '/var/lib/hare/cluster.yaml'
```

Comment:
Suggestion: `--file` or `--filename` instead of `--filepath`.

Example - docker-compose CLI:
```
Usage:
  docker-compose [-f <arg>...] [options] [COMMAND] [ARGS...]
  docker-compose -h|--help

Options:
  -f, --file FILE             Specify an alternate compose file
                              (default: docker-compose.yml)
```

Reply:
Done. I changed it to `--file`.
One question though. Isn't `--filename` sounds misleading considering that it is actually path not name.

Reply:
Perhaps it was not the best choice from my side. Up to this point I used to think that filename can contain path to the file (either relative or absolute). But it seems like it [was incorrect](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_170).